lib/ramble/ramble/pipeline.py

```diff
@@ -597,11 +604,26 @@ def _execute(self):
                 f.write(syaml.dump_config(repo_data))
 
         repo_path = os.path.join(self.workspace.named_deployment, self.object_repo_name)
-        object_types = ["applications", "modifiers", "packages"]
+        object_types = [
```

Comment:
Similar comment as before, but for this: https://github.com/GoogleCloudPlatform/ramble/blob/develop/lib/ramble/ramble/repository.py#L59

Reply:
Good point, I updated to use the `dir_name` from the type_definitions.